pkg/operator/certrotation/signer.go

```diff
@@ -37,6 +37,9 @@ type RotatedSigningCASecret struct {
 	// rotation on expiration only, but not interfere with the ordinary rotation controller.
 	RefreshOnlyWhenExpired bool
 
+	// Owner is an optional reference to add to the secret that this rotator creates.
+	Owner *metav1.OwnerReference
```

Comment:
Who is the typical owner, then? Why would you want the secret to be garbage collected if it disappears?

Please explain in the godoc or remove.

Reply:
I'm not sure how to explain that - the semantic meaning / reason for removal is up to the user.